src/Compiler/Directory.Packages.props

```diff
@@ -1,4 +1,9 @@
 ﻿<Project>
+  <PropertyGroup>
+    <_RoslynPackageVersion>4.9.2</_RoslynPackageVersion>
```

Comment:
Perhaps add a comment with the link https://learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#targeting-and-support-rules so people know how to (not) change this version